hubconf.py

```diff
@@ -6,27 +6,38 @@
 dependencies = ["torch"]
 
 from esm.pretrained import (
-    esm1_t34_670M_UR50S,
+    esm1_t6_43M_UR50S,
+    esm1_t12_85M_UR50S,
     esm1_t34_670M_UR50D,
+    esm1_t34_670M_UR50S,
     esm1_t34_670M_UR100,
-    esm1_t12_85M_UR50S,
-    esm1_t6_43M_UR50S,
     esm1b_t33_650M_UR50S,
-    esm_msa1_t12_100M_UR50S,
-    esm_msa1b_t12_100M_UR50S,
     esm1v_t33_650M_UR90S,
     esm1v_t33_650M_UR90S_1,
     esm1v_t33_650M_UR90S_2,
     esm1v_t33_650M_UR90S_3,
     esm1v_t33_650M_UR90S_4,
     esm1v_t33_650M_UR90S_5,
-    esm_if1_gvp4_t16_142M_UR50,
     esm2_t6_8M_UR50D,
     esm2_t12_35M_UR50D,
     esm2_t30_150M_UR50D,
     esm2_t33_650M_UR50D,
     esm2_t36_3B_UR50D,
     esm2_t48_15B_UR50D,
+    esm_if1_gvp4_t16_142M_UR50,
+    esm_msa1_t12_100M_UR50S,
+    esm_msa1b_t12_100M_UR50S,
+    esmfold_ipa_8M,
```

Comment:
As discussed async - let's rename to `esmfold_structure_module_only_*`, for clarity

Reply:
Done